docs/index.rst

```diff
@@ -0,0 +1,81 @@
+:relatedlinks: [Diátaxis](https://diataxis.fr/)
+
+.. _home:
+
+Testflinger
+============
+
+.. A single sentence that says what the product is, succinctly and memorably.
+Testflinger is a system for orchestrating the time-sharing of access to a pool of
+target machines.
+
+.. A paragraph of one to three short sentences, that describe what the product does.
+Each testflinger system consists of:
+
+* a microservce (called just testflinger) that provides an API to request jobs by placing
```

Comment:
As Matias said, probably more appropriate to just call it a "web service". I think we might want to fix this in the README also